runelite-client/src/main/java/net/runelite/client/plugins/friendlist/FriendListPlugin.java

```diff
@@ -109,6 +128,25 @@ else if (event.getScriptId() == ScriptID.IGNORE_UPDATE)
 		}
 	}
 
+	@Subscribe
+	public void onChatMessage(ChatMessage message)
+	{
+		if (config.showWorld() && message.getType() == ChatMessageType.LOGINLOGOUTNOTIFICATION)
+		{
+			// get the player name out of the notification
+			String name = message.getMessage().split(" ")[0];
```

Comment:
Did you test this on a player with a space in their name? It might actually work if the message uses non-breaking spaces for the player name, and also the `findByName` handles it, but I do not remember.

Reply:
I just tested it, works with spaces, but I changed the way of getting name to substring instead of split just to be safe.

Reply:
This isn't much better since if the old approach would break this new approach would also break if the user has ` has` in their name. So you could just use the old approach.